modules/cms-kit/src/Volo.CmsKit.Public.Web/CmsKitPublicWebModule.cs

```diff
@@ -61,6 +61,10 @@ public override void PostConfigureServices(ServiceConfigurationContext context)
         {
             Configure<RazorPagesOptions>(options =>
             {
+                if (GlobalFeatureManager.Instance.IsEnabled<PagesFeature>())
+                {
+                    options.Conventions.AddPageRoute("/CmsKit/Pages/Index", @"/pages/{pageUrl:minlength(1)}");
```

Comment:
Maybe we should change the route parameter to the `slug.` It will align with the domain.